src/indexer/app.rs

```diff
@@ -0,0 +1,13 @@
+extern crate clap;
+use self::clap::{Arg, App, SubCommand};
+
+pub fn make() -> App<'static, 'static> {
+    App::new("golden-retriever-indexer")
+        .version("0.0.1")
```

Comment:
Yes, definitely. Haskell for example generates a synthetic module that contains all sort of metadata about the package.